src/main/java/model/Player.java

```diff
@@ -0,0 +1,90 @@
+package model;
+
+
+import exception.InvalidMoneyAmountException;
+
+public class Player extends Person{
+    private final Wallet wallet;
+    private Bet bet;
+    private int playerNumber;
+
+    public Player(){
+        wallet = new Wallet(100);
```

Comment:
The wallet object is already initialized with default value of 100. Either call that constructor or remove the default constructor from the Wallet class